helm/prometheus-rules/templates/alerting-rules/silence-operator.rules.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: silence-operator
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: silences
+    rules:
+    - alert: "SilenceOperatorReconcileErrors"
+      annotations:
+        description: '{{`silence-operator controller {{ $labels.controller }} too many reconcile errors.`}}'
+        opsrecipe: "silence-operator-reconcile-errors/"
+      expr: |
+        avg_over_time(operatorkit_controller_errors_total{app="silence-operator"}[20m]) > 0
```

Comment:
`> 0` does it mean the alert will fire if there is only one error during the last 20 minutes ?

Reply:
For example: 
silence-operator encounter one error (due to whatever) but everything is running well after this trace, we are going to raise an alert and this alert will stay up during 1 hour before closing.
It's what I understand.
If it is, I think we can increase the threshold to be more cool...

Reply:
Which threshold? can you explain a bit more? :p

Reply:
I mean `> 0` could be replaced by `> X` 
No idea of the correct number of errors in 20m window

Reply:
I would just like this alert to fire only if an effective number of errors have occurred, you see what I mean ?

Reply:
Well, if it's greater than 0, an error occured :)

Reply:
Let's see how it work...